test/functional/change_stream.test.js

```diff
@@ -2597,233 +2599,236 @@ describe('Change Streams', function() {
           });
       });
     });
+  });
 
-    // For a ChangeStream under these conditions:
-    //   Running against a server >=4.0.7.
-    //   The batch is not empty.
-    //   The batch hasn’t been iterated at all.
-    //   The stream has iterated beyond a previous batch and a getMore command has just been executed.
-    // Expected result:
-    //   getResumeToken must return the postBatchResumeToken from the previous command response.
-    describe('for non-empty non-iterated batch where getMore has just been executed against server >=4.0.7', function() {
-      it('must return the postBatchResumeToken from the previous command response', function() {
-        const manager = new MockServerManager(this.configuration, {
-          aggregate: (function*() {
-            yield { numDocuments: 1, postBatchResumeToken: true };
-          })(),
-          getMore: (function*() {
-            yield { numDocuments: 1, postBatchResumeToken: true };
-          })()
-        });
-        let token;
-        const startAfter = manager.resumeToken();
-        const resumeAfter = manager.resumeToken();
-
-        return manager
-          .ready()
-          .then(() => {
-            return manager.makeChangeStream({ startAfter, resumeAfter }).next();
-          })
-          .then(() => {
-            manager.changeStream.cursor.once('response', () => {
-              token = manager.changeStream.resumeToken;
-            });
+  describe('tryNext', function() {
+    it('should return null on single iteration of empty cursor', {
+      metadata: { requires: { topology: 'replicaset', mongodb: '>=3.5.10' } },
```

Comment:
you can just say `>=3.6`, no need to codify development versions of the server once the feature has been released

Reply:
I used `3.5.10` to be consistent with the rest of the tests; all tests have now been updated to `>=3.6` for consistency.